src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label class="box">
+        <img
+           src="images/Search.svg"
+           alt="search icon"
+           class="search-icon"
+      >
+        <input
+        class="input"
+        type="search"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      </label>
+    </form>
+
+    <form
+     method="post"
+     action="http://google.pl"
+     data-qa="small"
+    >
+      <label class="box">
+        <img
+           src="images/Search.svg"
+           alt="search"
+           class="search-icon search-icon--small"
+      >
```

Comment:
fix indentation of closing tag
```suggestion
        <img
           src="images/Search.svg"
           alt="search"
           class="search-icon search-icon--small"
        >
```